src/engine/files.d.ts

```diff
@@ -3,6 +3,8 @@ declare module '*.png' {
   export default value;
 }
 
+declare module 'guitarLoop';
```

Comment:
This seems odd to have in the src/engine files.d.ts, can we move this another place?

Reply:
Perhaps a files.d.ts in the specific story root that can be referenced?

Reply:
This was a F2 rename-refactor issue, fixed.